provider.go

```diff
@@ -0,0 +1,124 @@
+package dynv6
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/libdns/libdns"
+)
+
+// Provider for dynv6 HTTP REST API
+type Provider struct {
+	// Token is required for authorization.
+	// You can generate one at: https://dynv6.com/keys
+	Token string
```

Comment:
Add JSON tag:
```suggestion
	Token string `json:"token,omitempty"
```
Allowing for easy JSON configuration :+1: